migrations/Version202203021737164734_taoBackOffice.php

```diff
@@ -0,0 +1,162 @@
+        'item/commonInteractions/extendedTextInteraction/hasMath' => 'hide',
+        'item/commonInteractions/hottextInteraction/disallowHTMLInHottext' => 'hide',
+        'item/customInteractions/customInteractions' => 'hide',
+        'item/customInteractions/likertScaleInteraction' => 'hide',
+        'item/customInteractions/liquidsInteraction' => 'hide',
+        'test/properties/item/itemSessionControl/showFeedback' => 'hide',
+        'test/properties/item/itemSessionControl/allowComment' => 'hide',
+        'test/properties/item/itemSessionControl/allowSkipping' => 'hide',
+        'test/properties/item/timeLimits' => 'hide',
+        'test/properties/section/itemSessionControl/showFeedback' => 'hide',
+        'test/properties/section/itemSessionControl/allowComment' => 'hide',
+        'test/properties/section/itemSessionControl/allowSkipping' => 'hide',
+        'test/properties/section/timeLimits' => 'hide',
+        'test/properties/testPart/itemSessionControl' => 'hide',
+        'test/properties/testPart/timeLimits' => 'hide',
+        'test/properties/test/timeLimits' => 'hide',
+        'delivery/taoLocal' => 'hide'
+    ];
+
+    const EXISTING_CONFIG_MAP = [
+        'tao' => [
+            'client_lib_config_registry' => [
+                'item/image/mediaAlignment' => [
+                    'ui/image/ImgStateActive' => ['mediaAlignment'],
+                ],
+                'test/item/BRS' => [
+                    'taoQtiTest/controller/creator/views/item' => ['BRS'],
+                ],
+                'item/commonInteractions/extendedTextInteraction/hasMath' => [
+                    'taoQtiItem/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question' => ['hasMath'],
+                ],
+                'item/commonInteractions/hottextInteraction/disallowHTMLInHottext' => [
+                    'taoQtiItem/qtiCreator/widgets/interactions/hottextInteraction/states/Question'=> ['disallowHTMLInHottext'],
+                ]
+            ]
+        ],
+        'taoQtiItem' => [
+            'qtiCreator' => [
+                'item/multiColumn' => [
+                    'multi-column' => []
+                ],
+                'item/scrollableMultiColumn' => [
+                    'scrollable-multi-column' => []
+                ],
+                'item/perInteractionRp' => [
+                    'perInteractionRp' => []
+                ],
+            ]
+        ]
+    ];
+
+    public function up(Schema $schema): void
+    {
+        $resultConfig = self::DEFAULT_CONFIG;
+        $backOfficeExtension = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoBackOffice');
+        if($backOfficeExtension === null) {
+            throw new \Exception(sprintf('Cannot find %s extension', 'taoBackOffice'));
+        }
+        $existingConfig = $backOfficeExtension->getConfig(self::GLOBAL_UI_CONFIG_NAME);
+        if($existingConfig !== false) {
```

Comment:
It seems your PHPStorm is misconfigured and not following the PSR-12 code style. Can you fix it, please?

Reply:
fixed